### PR TITLE
Update NLRModels.md

### PR DESCRIPTION
**Pull request by CezaryMarcjan**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: Orchestrator/docs/NLRModels.md [hunk 1]
```diff
@@ -2,16 +2,24 @@
...
 
 ## Models
+The public preview of Orchestrator includes the following prebuilt language models available in [versions repository][2].
+
+### pretrained.20200924.microsoft.dte.00.03.en.onnx
+This is a fast and small base model with sufficient accuracy but if the accuracy and not speed and memory size is critical then consider other options. It is a 3-layer pretrained BERT model optimized for conversation for example-based use (KNN).
 
-**TBD**: Describe models in [versions repository][2].
+### pretrained.20200924.microsoft.dte.00.06.en.onnx
+This is A high quality base model that strikes the balance between size, speed and accuracy. It is a 6-layer pretrained BERT model optimized for conversation for example-based use (KNN).
```
Comment (tsuwandy) [inline Orchestrator/docs/NLRModels.md:16]: “a high quality...” (lowercase a).  Should we keep the knn at the end of the description? It needs its own description for non ML folks.

**Review by daveta — APPROVED** [review]

File: Orchestrator/docs/NLRModels.md [hunk 2]
```diff
@@ -2,16 +2,24 @@
...
 
+### pretrained.20200924.microsoft.dte.00.12.en.onnx
+This is a highest quality base model but is larger and slower than other options. It is a 12-layer pretrained BERT model optimized for conversation for example-based use (KNN).
```
Comment (tsuwandy) [inline Orchestrator/docs/NLRModels.md:19]: Not sure about the knn at the end of the description of all models.  Might want to link it to a page describing the term?

**Review by tsuwandy — APPROVED** [review]

File: Orchestrator/docs/NLRModels.md [hunk 3]
```diff
@@ -2,16 +2,24 @@
...
+Prebuilt language models have been trained towards more sophisticated tasks for both monolingual as well as multilingual scenarios. In public preview only English models are made available.
 
 ## Models
+The public preview of Orchestrator includes the following prebuilt language models available in [versions repository][2].
+
+### pretrained.20200924.microsoft.dte.00.03.en.onnx
+This is a fast and small base model with sufficient accuracy but if the accuracy and not speed and memory size is critical then consider other options. It is a 3-layer pretrained BERT model optimized for conversation for example-based use (KNN).
 
-**TBD**: Describe models in [versions repository][2].
+### pretrained.20200924.microsoft.dte.00.06.en.onnx
```
Comment (daveta) [inline Orchestrator/docs/NLRModels.md:15]: We might mention this is the "default".

Comment (scheyal) [reply Orchestrator/docs/NLRModels.md:15]: Agree. mention it is the default

File: Orchestrator/docs/NLRModels.md (hunk 2 above)
Comment (scheyal) [reply Orchestrator/docs/NLRModels.md:19]: Add reference to KNN, spell it out? What is KNN Kohonen Neural Net, k-nearest neighbors...(and link to https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm)

**Review by scheyal — APPROVED** [review]
Added a few minor changes (remove extra bullet, link to knn)

File: Orchestrator/docs/NLRModels.md (hunk 3 above)
Comment (CezaryMarcjan) [reply Orchestrator/docs/NLRModels.md:15]: good idea. done.

File: Orchestrator/docs/NLRModels.md (hunk 2 above)
Comment (CezaryMarcjan) [reply Orchestrator/docs/NLRModels.md:19]: good idea. done.

**Pull request merged**